src/test/java/racingcar/RacingCarTest.java

```diff
@@ -3,20 +3,22 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.Assertions.*;
 
 class RacingCarTest {
     private static final int ATTEMPT_NUMBER = 4;
     private static final int ATTEMPT_NUMBER_ZERO = 0;
+    private static final String NAME = "happy";
+    private static final CarMover RANDOM_NUMBER_CAR_MOVER = new RandomNumberMover();
```

Comment:
테스트용 Mover를 implmement 해주는것도 좋을 것 같아요!

Reply:
```suggestion
    private static final CarMover ALWAYS_CAR_MOVER = new CarMover(){
        @Override
        public boolean movable() {
            return true;
        }
    }
```
이렇게 항상 true인 Mover를 반환할 수 도 있는거죠!

혹은 경우에 따라서 무조건 false인 Mover도 반환할 수 있고, true와 false를 번갈아 가면서 내뱉는 Mover도 반환할 수 있습니다.

이게 바로 전략패턴의 핵심입니다!

Reply:
지난번 리뷰 때 고정 이동조건이라고 말씀주셨던 부분이 이 내용이었군요 ..!
랜덤숫자 이동조건을 고정시켜 테스트하는 것으로 잘못 이해했네요 아직 전략패턴에 대해 공부가 부족한가봅니다 ㅜㅜ
고정된 결과값을 내려주거나 반복적으로 값을 내려주는 테스트용 Mover 생성해서 해보겠습니다 !